src/modules/decklink/consumer/decklink_consumer.cpp

```diff
@@ -518,9 +708,9 @@ struct decklink_consumer final : public IDeckLinkVideoOutputCallback
                                     nb_samples);
             }
 
-            std::shared_ptr<void> image_data = create_aligned_buffer(decklink_format_desc_.size);
+            std::shared_ptr<void> image_data = allocate_frame_data(decklink_format_desc_, config_.hdr);
 
-            schedule_next_video(image_data, nb_samples, video_scheduled_);
```

Comment:
Well... It just sets the color space metadata on the empty frames in the initial buffer... Ideally, I guess this should match the color space of the channel. But then that would have to be propagated down here, and I can not see it making any difference.

Reply:
I'm expecting it to do something based on `config.hdr`.

I'm not sure if defaulting to 2020 will work, because not all decklinks support 2020. And for those that do, will changing colorspace mid-stream work seamlessly, or will the decklink be upset about that? So at least making it do 709 or 2020 based on `config.hdr` would eliminate a lot of the risk until you opt into the hdr mode